src/SurfaceFluxes.jl

```diff
@@ -348,28 +393,181 @@ end
 
 ### Generic terms
 
+function b_star_from_ρ(param_set, 
```

Comment:
I think all b_star functions below should receive as inputs the thermodynamic states at two levels and get from them whatever quantity is required (T,θ,ρ)